### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by killerheman**

### CODE REVIEW

File: app/Http/Controllers/Backend/BranchController.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+
+        $branchd=Branch::where('deleted_at',Null)->get();
```
Comment (softpromani) [inline app/Http/Controllers/Backend/BranchController.php:20]: remove where condition and import facades for soft delete

Comment (softpromani) [inline app/Http/Controllers/Backend/BranchController.php:20]: remove where condition and use softdelete libraray

**Review by softpromani — CHANGES_REQUESTED** [review]
remove  where condition

File: app/Http/Controllers/Backend/BranchController.php (hunk 1 above)
Comment (DeepakGautam420) [reply app/Http/Controllers/Backend/BranchController.php:20]: All done sir

**Review by softpromani — APPROVED** [review]

**Pull request merged**